text/0000-general-fund.md

```diff
@@ -88,6 +88,14 @@ Everyone is welcome to weigh in on the merits of a funding request in the forum
 * Be specific, professional and thorough with your proposal.
 * Be respectful and professional in your response to any feedback and criticism on the proposal thread 
 
+# Payout Process
+
+* At the time of payout, an amount of Bitcoin equivalent to the funding amount is paid out to an address provided by the funding requestor only.
+* The payout address should be provided to the core team by the requestor via a secure, verifiable channel (with cryptographic proof of identity if deemed necessary).
+* The exchange rate between BTC and the target currency is determined by the exchange rate at the time of the funding transaction, without exception.
```

Comment:
...any exchange rate between the currency paid out from the general fund, and the currency the funding request is raised in...

or similar.

Reply:
Bump on this comment